build.js

```diff
@@ -3,6 +3,17 @@ const StyleDictionary = require('style-dictionary')
 
 const {fileHeader, formattedVariables} = StyleDictionary.formatHelpers
 
+//-----
+// functions to be extracted
+// TODO: extract to a separate files
+
+const pathToKebabCase = token => token.path.join('-')
+
+const pathToDotNotation = token => token.path.join('.')
+
+const capitalize = string => string[0].toUpperCase() + string.slice(1)
+const pathToPascalCase = token => token.path.map(tokenPathItems => capitalize(tokenPathItems)).join('')
```

Comment:
being pedantic, can we add a newline here like the others please? 😅 

```suggestion

const pathToPascalCase = token => token.path.map(tokenPathItems => capitalize(tokenPathItems)).join('')
```